lib/convertUnits.ts

```diff
@@ -0,0 +1,32 @@
+import { BigNumber } from 'bignumber.js'
```

Comment:
I'd say that this file has a problem with public interface not being clearly visible. 

Everything is exported but only some of the things are used (or tested) by external modules.